src/org/labkey/test/pages/core/admin/BaseSettingsPage.java

```diff
@@ -351,12 +398,19 @@ public enum DATE_FORMAT
     {
         yyyy_MM_dd("yyyy-MM-dd"),
         yyyy_MMM_dd("yyyy-MMM-dd"),
+        yyyy_MM("yyyy-MM"),
         dd_MMM_yyyy("dd-MMM-yyyy"),
         dd_MMM_yy("dd-MMM-yy"),
+        dd_MM_yyyy("dd-MM-yyyy"),
         ddMMMyyyy("ddMMMyyyy"),
         ddMMMyy("ddMMMyy"),
+        MMddyyyy("MM/dd/yyyy"),
+        MM_dd_yyyy("MM-dd-yyyy"),
+        MMMM_dd_yyyy("MMMM dd yyyy"),
```

Comment:
is `MMMM_dd_yyyy` used in any test? It's kinda special in that it contains space. I think it's good to check setting date/datetime field that uses this type and verify it's persisted / parsed correctly.

Reply:
I have changed a couple of tests in LKS and LKSM to use `MMMM_dd_yyyy`